server/engine/src/main/antlr4/org/eclipse/lsp/cobol/implicitDialects/sql/Db2SqlExecLexer.g4

```diff
@@ -900,50 +881,33 @@ SQLLINECOMMENT
 // treat all the non-processed tokens as errors
 ERRORCHAR : . ;
 
-ZERO_DIGIT: '0';
-
 fragment HEXNUMBER :
 	X '"' [0-9A-Fa-f]+ '"'
 	| X '\'' [0-9A-Fa-f]+ '\''
 ;
 
-fragment NULLTERMINATED :
-	Z '"' (~["\n\r] | '""' | '\'')* '"'
-	| Z '\'' (~['\n\r] | '\'\'' | '"')* '\''
-;
-
-fragment STRINGLITERAL :
-	'"' (~["\n\r] | '""' | '\'')* '"'
-	| '\'' (~['\n\r] | '\'\'' | '"')* '\''
-;
-
-fragment UNTRMSTRINGLITERAL :
-	'"' (~["\n\r] | '""' | '\'')*
-	| '\'' (~['\n\r] | '\'\'' | '"')*
-;
-
-fragment DBCSLITERAL :
-	[GN] '"' (~["\n\r] | '""' | '\'')* '"'
-	| [GN] '\'' (~['\n\r] | '\'\'' | '"')* '\''
-;
-
 fragment BXNUMBER :
-	B X '"' [0-9A-F]+ '"'
-	| B X '\'' [0-9A-F]+ '\''
+	B X '"' [0-9A-Fa-f]+ '"'
+	| B X '\'' [0-9A-Fa-f]+ '\''
 ;
 
 fragment GRAPHICUNICODE :
-	U X '"' [0-9A-F]+ '"'
-	| U X '\'' [0-9A-F]+ '\''
+	U X '"' [0-9A-Fa-f]+ '"'
+	| U X '\'' [0-9A-Fa-f]+ '\''
 ;
 
 fragment GRAHICCHAR :
-	G X '"' [0-9A-F]+ '"'
-	| G X '\'' [0-9A-F]+ '\''
+	G X '"' [0-9A-Fa-f]+ '"'
+	| G X '\'' [0-9A-Fa-f]+ '\''
+;
+
+fragment STRINGLITERAL :
+	'"' (~["\n\r] | '""' | '\'')* '"'
+	| '\'' (~['\n\r] | '\'\'' | '"')* '\''
 ;
 
 fragment
- OCT_DIGIT        : [0-8] ;
+ OCT_DIGIT        : [0-9] ;
  fragment DIGIT: OCT_DIGIT | [9];
```

Comment:
Assuming `OCT_DIGIT` was meant to be an octal digit, it should have been 0-7.
Also `OCT_DIGIT` and `DIGIT` are now the same fragment.

Reply:
Thanks ! updated.